### ISSUE TITLE
Lack of visual indication of keyboard focus on some elements

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/videojs.com#92]
Some elements which receive keyboard focus (such as the Brightcove logo on the home page) do not display a focus outline.
